src/main/resources/org/jenkinsci/plugins/nomad/NomadSlaveTemplate/config.jelly

```diff
@@ -16,6 +16,10 @@
             <f:textbox/>
         </f:entry>
 
+        <f:entry title="${Constraints}" field="constraints" description="comma separated LTarget,RTarget,Operand">
```

Comment:
Good point on the comma, I should use a different delimiter. It is hard to decide the "best" way to input the data into the field, as the three main challenges are:

1.) Need to input N groups of 3 items in a single textbox
2.) Textbox filters out certain characters (i.e. ${}, which Nomad requires in the LTarget for certain constraints)
3.) Cannot use comma as a separator

I'll wait a few days to see If there are some good suggestions for how to input the text into this field, I can implement them. Otherwise, I'll go ahead and come up with something on my own (might not be very user friendly or intuitive, though.)